src/main/scala/nl/knaw/dans/api/sword2/DepositHandler.scala

```diff
@@ -121,6 +124,17 @@ object DepositHandler {
     }
   }
 
+  def checkBaseDir()(implicit id: String, baseDir: File): Try[Unit] = Try {
```

Comment:
Well, I thought that it is also proper Scala to wrap a method in a `Try`.
As you have seen, I use in more places in this code exceptions with the idea that the program goes directly to the main method where the exception will be caught.  I have noticed that also in other places you prefer to have Failure rather than throw exception. I am wondering, is that a matter of taste, or do you find that throwing an exception is really bad?